install/updateVersion.sh

```diff
@@ -196,8 +199,9 @@ function update_helm_version() {
 
   execute_sed "s|{CA_HUB}|${CA_HUB}|"       values.yaml.tmpl
   execute_sed "s|{CA_TAG}|${CA_TAG}|"       values.yaml.tmpl
-  execute_sed "s|{PROXY_HUB}|${PILOT_HUB}|" values.yaml.tmpl
+  execute_sed "s|{PROXY_HUB}|${PROXY_HUB}|" values.yaml.tmpl
```

Comment:
`PROXY_HUB` is never defined in `istio.VERSION`. Also if you check `updateVersion.sh, it's never used, each seed which target is `{PROXY_HUB}` use `${PILOT_HUB}` variable as value. No idea why it was defined if it's never used or if it is a mistake how is being used right now.

cc @linsun

Reply:
This is an artifact of our previous (before mono-repo) code structure. The istio-proxy container is produced in part from code that lives in Pilot's codebase. Because of this, the scripts to generate and push the istio-proxy container live in Pilot's repo too; for that reason all of the scripting uses Pilot's info when referring to the istio-proxy container.

With the new monorepo, the (longish term) plan is to migrate that kind of stuff up to the root of the Istio project. When than happens, we'll likely want the stronger divide between the istio-proxy container and the other artifacts from Pilot.